src/main/resources/application.yml

```diff
@@ -21,6 +21,18 @@ spring:
     user: ${spring.datasource.username}
     password: ${spring.datasource.password}
 
+  mail:
+    host: smtp.gmail.com
+    port: 587
+    username: hatim.slimani.19@gmail.com
+    password: qihixwerqwotokir
```

Comment:
putting credentials directly in application.yml is risky because they can easily end up exposed through version control or logs instead of hardcoding usernames and passwords it’s better to load them from environment variables or a secret manager you can also include a .env.example file in the repo to show which keys are needed without revealing the real values These credentials should be rotated right away since they might already be compromised